rails-new.md

```diff
@@ -21,19 +21,21 @@ In your new directory make a `docker-compose.yml` file and paste in:
 version: "3"
 services:
   app:    
-    image: brandnewbox/bnb-ruby:2.7-postgresql
+    image: brandnewbox/bnb-ruby:3.1-postgresql
     command: bundle exec puma -C config/puma.rb
+    environment:
+      - DATABASE_URL=postgres://postgres:monkey@postgresservice:5432
     volumes:
       - .:/app:cached
       - bundle_cache:/usr/local/bundle
     ports:
       - "3000:3000"
     depends_on:
-      - postgres
-  postgres:
+      - postgresservice
```

Comment:
I haven't seen `postgresservice`. Is this doing the same?  I'm also assuming the password doesn't matter here, but why monkey?

Reply:
This is intentional. Mostly trying to help folks parse what's going on. 

Previously: the service, the username, the actual name of the application were all the same: `postgres`

By introducing differences here i'm trying to help first-time users map between things. Like "Oh... postgresservice is in two places... they must map to each other. And password isn't just password.. it's monkey, which is set down here."